dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -74,6 +79,20 @@ export const TopicFilterBank = ({
 	filterKeyEvents = false,
 }: Props) => {
 	const palette = decidePalette(format);
+
+	const selectedTopic = selectedTopics?.[0];
+	const topFiveTopics = availableTopics.slice(0, 5);
+
+	if (selectedTopic) {
+		const selectedIndex = availableTopics.findIndex(
```

Comment:
could move this inside the if statement below as we only need it if selectedIntex > 4

Reply:
I may not have fully understood what you mean. Here we're getting the `selectedIndex` and in the if statement below, it's using the value of the `selectedIndex` as a condition.